src/main/java/com/purbon/kafka/topology/BuilderCLI.java

```diff
@@ -128,8 +158,47 @@ public void run(String[] args) throws IOException {
     String topology = cmd.getOptionValue(TOPOLOGY_OPTION);
     Map<String, String> config = parseConfig(cmd);
 
-    processTopology(topology, config);
-    System.out.println("Kafka Topology updated");
+    if (config.get(EXTRACT_OPTION).equals("true")) {
+      config.put("bootstrap.servers", config.get(BROKERS_OPTION));
+      try (AdminClient adminClient = AdminClient.create((Map) config)) {
```

Comment:
AdminClient is abstracted internally, how would you for example recover ACLs when using RBAC?

Reply:
Agreed, to current implementation doesn't cover this case.